events_task/lib/my_app/content/content.ex

```diff
@@ -48,23 +35,49 @@ defmodule MyApp.Content do
     Repo.all(Confirmation)
   end
 
-  def list_user_confirmed_event_ids(user) do
+  def list_confirmations_by_user(id) do
+    from(c in Confirmation,
+      where: c.user_id == ^id,
+      select: c
```

Comment:
If you are doing `from(c in Confirmation, ...)`, then `select: c` is the default - you can remove it.